tests/e2e/vc_reboot_volume_lifecycle.go

```diff
@@ -90,7 +90,8 @@ var _ bool = ginkgo.Describe("Verify volume life_cycle operations works fine aft
 			scParameters[scParamStoragePolicyID] = profileID
 			// create resource quota
 			createResourceQuota(client, namespace, rqLimit, storagePolicyName)
```

Comment:
You need to add quota for gc scenario as well?

Reply:
done